packages/react-scripts/template/README.md

```diff
@@ -18,6 +18,7 @@ You can find the most recent version of this guide [here](https://github.com/fac
 - [Importing a Component](#importing-a-component)
 - [Adding a Stylesheet](#adding-a-stylesheet)
 - [Post-Processing CSS](#post-processing-css)
+- [Adding CSS Preprocessor (SASS, LESS etc.)](#adding-css-preprocessor-sass-less-etc)
```

Comment:
Ok, this is just a pure and not serious comment: when I saw you change the name from "Less" to "LESS", I'm wondering what is the official term, so I take a look at both website and realize that they actually use the lower case :)) It is "Sass" and "Less". I previously thought that it is "Less" because it is not a shorthand, while "SASS" is acutally short for "Syntactically Awesome Stylesheets".